pkg/cli/sql.go

```diff
@@ -353,9 +353,13 @@ func preparePrompts(dbURL string) (fullPrompt string, continuePrompt string) {
 	// continued statement prompt is: "        -> "
 	fullPrompt = dbURL
 	if parsedURL, err := url.Parse(dbURL); err == nil {
+		username := ""
```

Comment:
Defaulting to an empty username doesn't seem right. We should either return an error if no user name is given or default to a more useful value (the value of `--user`, which defaults to `root`?)